pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "repl-nix-landmarkexplorer"
+name = "app"
```

Comment:
The removal of the `replit.nix` suggests a shift away from Replit-specific configurations. Be sure to provide alternative setup steps or make it clear in the documentation how to handle dependencies like PostgreSQL and OpenSSL without this Nix file.